src/corelocation.cs

```diff
@@ -293,18 +285,24 @@ partial interface CLLocationManager {
 
 		[NoWatch][NoTV]
 		[iOS (6,0)]
+		[Deprecated (PlatformName.iOS, 13,0, message: "Not used anymore. Call will not have any effect.")]
+		[Unavailable (PlatformName.UIKitForMac)][Advice ("This API is not available when using UIKit on macOS.")]
 		[Export ("allowDeferredLocationUpdatesUntilTraveled:timeout:")]
 		void AllowDeferredLocationUpdatesUntil (double distance, double timeout);
 
 		[NoWatch][NoTV]
 		[iOS (6,0)]
+		[Deprecated (PlatformName.iOS, 13,0, message: "Not used anymore. Call will not have any effect.")]
+		[Unavailable (PlatformName.UIKitForMac)][Advice ("This API is not available when using UIKit on macOS.")]
 		[Export ("disallowDeferredLocationUpdates")]
 		void DisallowDeferredLocationUpdates ();
 #endif
 
 		[NoWatch][NoTV]
 		[iOS (6,0)]
 		[Mac (10,9)]
+		[Deprecated (PlatformName.iOS, 13,0, message: "Not used anymore. It will always return .false'.")]
```

Comment:
Just wondering if ".false" is a typo?

Reply:
yes it should have been `'false'`
good catch !